### PR TITLE
Add consumables

### PR DESCRIPTION
**Pull request by OhPointFive**
This adds a consumables page to the documentation, mostly copied & modified from the projectiles page.

### CODE REVIEW

File: docs/modules/gear/consumables.mdx [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+        </td>
+        <td>
+          <span className="badge badge--danger">Alias</span>
+          Alternative to <label>action</label>, encouraged if the consumable applies a kit.
+        </td>
+        <td>
+          <a href="/docs/modules/gear/kits">Kit</a>
+        </td>
+        <td></td>
+      </tr>
```
Comment (Pablete1234) [inline docs/modules/gear/consumables.mdx:92]:
```suggestion
```
I'd suggest *against* even listing kit as an alternative, we don't want maps to be using both, we want it to be standardized. Older features were made *before* action even existed, and kit was a thing, and so it has to stay as legacy functionality, but in consumables i'd avoid encouraging it.

Others can feel free to give their opinion on this one too, but i think we should not encourage 2 standards

File: docs/modules/gear/consumables.mdx [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+      <tr>
+        <td>
+          <label>on</label>
+        </td>
+        <td>
+          How the consumable is consumed.
```
Comment (Pablete1234) [inline docs/modules/gear/consumables.mdx:98]:
```suggestion
        <td>
          <span className="badge badge--danger">Required</span>
          How the consumable is consumed.
```
Pretty sure this was made required, i'd also add a clarifying comment saying that while currently the only supported action is eat, in the future more actions (eg: clicking) may be supported

File: docs/modules/gear/consumables.mdx [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+    </tbody>
+  </table>
+</div>
+
+### Examples
+
+```xml
+<!-- Create the consumable "template" -->
+<consumables>
+    <consumable id="fast-apple" kit="speed-kit"/>
```
Comment (Pablete1234) [inline docs/modules/gear/consumables.mdx:128]:
```suggestion
    <consumable id="fast-apple" kit="speed-kit" on="eat"/>
```
pretty sure on is required

File: docs/modules/gear/consumables.mdx [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+<kits>
+    <kit id="spawn">
+        <item slot="1" amount="5" consumable="porkchop-that-says-yum">pork</item>
+    </kit>
+</kits>
+<!-- Define the action the consumable gives you -->
+<actions>
+    <action id="say-yum" scope="player">
+        <message text="Yum!"/>
+    </action>
```
Comment (Pablete1234) [inline docs/modules/gear/consumables.mdx:157]:
```suggestion
    <message id="say-yum" text="Yum!"/>
```
Pretty certain you don't need the wrapping action as you have a single child

**Review by Pablete1234 — CHANGES_REQUESTED** [review]

File: docs/modules/gear/consumables.mdx (hunk 1 above)
Comment (CoWinkKeyDinkInc) [reply docs/modules/gear/consumables.mdx:92]: Yes, we want to move most things from Kit to Action eventually, the documentation should reflect this.

**Comment by CoWinkKeyDinkInc** [comment]
I agree with Pablo's suggestions. I would also change this line in `docusaurus.config.js` to announce the new changes (remove payload), I can do this myself if you want.
```
announcementBar: {
      id: 'new_features',
      content:
        'New Features: <a href="/docs/modules/blocks/structures">Structures</a>, <a href="/docs/modules/blocks/enderchests">Ender Chests</a> and <a href="/docs/modules/objectives/payload">Payload Gamemode</a>',
      backgroundColor: '#fafbfc',
      textColor: '#091E42',
      isCloseable: true,
    },
```

**Comment by CoWinkKeyDinkInc** [comment]
Replaced by #116

**Pull request closed without merging**